### PR TITLE
updated symfony dep to allow all 2.x versions

### PR DESCRIPTION
**Pull request by shieldo**
It should be possible for this bundle to be compatible for all subsequent 2.x releases, so this change declares the dependencies as such.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "symfony/symfony": ">=2.1.0,<2.3-dev",
+        "symfony/symfony": ">=2.1.0,<3.0",
```
Comment (stof) [inline composer.json]: you should mark it as `~2.1` IMO

**Comment by shieldo** [comment]
@stof I've modified the declaration as I agree `~2.1` is better.

**Comment by shieldo** [comment]
Any chance of a merge?

**Comment by shieldo** [comment]
Thanks Dennis!

**Pull request merged**